Experiments/Experiments/FeatureFlag.swift

```diff
@@ -149,7 +149,7 @@ public enum FeatureFlag: Int {
     case shareProductAI
```

Comment:
These errors are still being reported on CI:

- `WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift:1365:52`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayCompleteViewController.swift:135:32`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayInformationViewController.swift:199:28`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `WooCommerce/Classes/ViewRelated/Search/SearchViewController.swift:341:21`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `Yosemite/Yosemite/Model/Mocks/Graphs/ScreenshotsObjectGraph.swift:13:26`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `Yosemite/Yosemite/Model/Mocks/Graphs/ScreenshotsObjectGraph.swift:14:31`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `Yosemite/Yosemite/Model/Mocks/Graphs/ScreenshotsObjectGraph.swift:15:31`: Localized strings should include a description giving context for how the string is used. (localization_comment)

Reply:
🤔 interesting to see that CI considers them error when they are but warnings on my end.

Reply:
Actually, not interesting at all, given CI uses `--strict` 😅